### PR TITLE
Add smaller version as `dcflights`

### PR DESCRIPTION
**Pull request by juliasilge**
This adds a scaled down version of the original dataset (only July 2021, only DCA departures, only top 10 destinations) for us to use the first day of the workshops:

``` r
library(dcflights21)
data("dcflights")

dplyr::glimpse(dcflights)
#> Rows: 2,703
#> Columns: 8
#> $ airline     <chr> "American Airlines Inc.", "American Airlines Inc.", "Ameri…
#> $ flight      <fct> 348, 557, 564, 421, 323, 135, 372, 224, 62, 665, 344, 1127…
#> $ destination <fct> CLT, DFW, ORD, ATL, ATL, BNA, DTW, MIA, CLT, ORD, DFW, BOS…
#> $ date        <date> 2021-07-01, 2021-07-01, 2021-07-01, 2021-07-01, 2021-07-0…
#> $ hour        <dbl> 5, 5, 5, 6, 6, 6, 6, 6, 6, 6, 7, 7, 7, 7, 7, 7, 7, 8, 8, 8…
#> $ visibility  <dbl> 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10…
#> $ duration    <dbl> 57, 158, 105, 74, 76, 89, 62, 141, 58, 107, 151, 58, 166, …
#> $ dep_delay   <dbl> -4, -5, -4, -4, 0, 0, -5, -3, -6, -4, -6, -3, -2, 0, -6, 1…
```

<sup>Created on 2022-05-07 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

### CODE REVIEW

File: R/dcflights21.R [hunk 1]
```diff
@@ -13,14 +13,48 @@
 #'   \item{origin, destination}{Origin and destination airports.}
 #'   \item{date}{Date of departure.} 
 #'   \item{hour}{Time of scheduled departure by hour}
-#'   \item{dep_delay}{Departure delay, in
-#'     minutes. Negative values represent early departures/arrivals.}
 #'   \item{plane}{Plane tail number, giving a unique ID for each plane.} 
-#'   \item{air_time}{Scheduled amount of time spent in the air, in minutes.} 
-#'   \item{duration}{Distance between airports, in miles.} 
+#'   \item{distance}{Distance between airports, in miles.} 
+#'   \item{duration}{Scheduled amount of time spent in the air, in minutes.}
```
Comment (juliasilge) [inline R/dcflights21.R:18]: @simonpcouch Notice that I changed this in the original dataset. Can you confirm this is right now?

Comment (simonpcouch) [reply R/dcflights21.R:18]: Ah, nice catch!

File: R/dcflights21.R [hunk 2]
```diff
@@ -13,14 +13,48 @@
...
+
+
+#' Flights data
+#'
+#' Data for flights that departed from the Washington DC airport (Reagan 
+#' Internation) during July 2021 and arrived at one of the top 10 connecting
+#' airports.
+#'
+#' @name dcflights
+#' @aliases flights
```
Comment (simonpcouch) [inline R/dcflights21.R]: Seeing a warning on Actions about duplicated aliases—could you drop this line or the analogous one for the larger dataset and redocument? Thumbs up from me if you'd like them to share an `@rdname`, though.🙂

Comment (juliasilge) [reply R/dcflights21.R]: I think I'd rather keep them in separate man files so that the first day, folks can just focus on this dataset only.

**Comment by simonpcouch** [comment]
Perfect, thanks!🚀

**Pull request merged**